app/forms/publish/school_form.rb

```diff
@@ -41,8 +42,11 @@ def compute_fields
     end
 
     def location_name_unique_to_provider
-      sibling_sites = provider.sites - [site]
-
+      sibling_sites = if site_type == 'study_site'
```

Comment:
think we can just do `study_site?` here?

Reply:
Yep, good spot.